stages/org.osbuild.bootc.install-to-filesystem.meta.json

```diff
@@ -7,12 +7,16 @@
     "mounted in the \"mounts\" path.",
     "Buildhost commands used: bootc"
   ],
-  "capabilities": ["CAP_MAC_ADMIN"],
+  "capabilities": [
+    "CAP_MAC_ADMIN"
+  ],
   "schema_2": {
     "inputs": {
       "type": "object",
       "additionalProperties": false,
-      "required": ["images"],
+      "required": [
+        "images"
+      ],
```

Comment:
Nitpick: these changes should go to the previous commit, which changed the formatting.